src/pages/components/pagination/usage.mdx

```diff
@@ -54,55 +59,377 @@ control for navigating to the next or previous page.
+
+<Row>
+<Column colLg={12}>
+
+![Example of the pagination nav variant in a UI](images/pagination-usage-pagination-nav-in-context.png)
+
+</Column>
+</Row>
+
+<Caption>Example of the pagination nav variant in a UI</Caption>
+
+### States
+
+The pagination nav variant has the ghost icon button component and the
+breadcrumb overflow button component and nested within it and inherits their
+component states. For more information on these states, refer to the
+[select](https://carbondesignsystem.com/components/select/style/) and
+[breadcrumb overflow button](https://carbondesignsystem.com/components/breadcrumb/style/)
+style tab guidance.
+
+### Interactions
+
+#### Mouse
+
+- Clicking on a button number will navigate you to that page.
+- Clicking on a previous or next caret button will navigate you to the previous
+  and next pages of the one you are currently viewing.
+- Clicking on an overflow ellipsis button will open a select menu to choose a
+  page to navigate to. Upon clicking on a page option from the menu, you will
+  navigate to that page.
+
+#### Keyboard
+
+- The icon button receives focus and you can move between buttons by pressing
+  the `Up` or `Down` arrows.
+- The page number buttons are activated by pressing `Space` or `Enter`.
+- The previous and next buttons are activated by pressing `Space` or `Enter`.
+
+### Responsive behavior
+
+In small screens, the pagination nav variant can provide an ellipsis button to
+indicate there are more pages to navigate to within the ellipsis button menu.
+
 <Row>
 <Column colLg={8}>
 
-![Example of pagination controls on data table](images/pagination-usage-1.png)
+![Pagination nav variant responsive behavior in large versus small breakpoints](images/pagination-usage-responsive-behavior-pagination-nav.png)
 
 </Column>
 </Row>
 
-#### Items per page
+<Caption>
+  Pagination nav variant responsive behavior in large versus small breakpoints
+</Caption>
 
-Use an inline select within the pagination bar so the user can change the amount
-of data displayed per page.
+## Clickable areas
+
+Every button in the pagination nav variant is clickable. Two ghost icon buttons
+take you to the previous or next page. The rest of the page number buttons take
+you to that particular page.
 
 <Row>
 <Column colLg={8}>
 
-![Example of items per page on data table](images/pagination-usage-2.png)
+![Clickable areas within the pagination nav variant](images/pagination-usage-clickable-areas-pagination-nav.png)
 
 </Column>
 </Row>
 
+<Caption>Clickable areas within the pagination nav variant</Caption>
+
+## Modifiers
+
+### Page looping
+
+For the pagination nav variant, instead of turning off the previous or next
+caret buttons when you reach either the first or last page of the pagination
+component, you can enable the option to loop through the available pages
+continuously.
+
+### Page naming
+
+The select components text is in number form by default. It shows the number of
```

Comment:
This should be possessive, right? 
```suggestion
The select component's text is in number form by default. It shows the number of
```

Reply:
Also is this duplicate content with the content section?

Reply:
Yeah it is...I was not sure if I should mention this in both the content section and modifier section or which place would be more appropriate.

Reply:
I think it's fine to have it in two places. Tabs has it and I think it makes sense to have it in the [Content](https://carbondesignsystem.com/components/tabs/usage/#content) and [Modifier](https://carbondesignsystem.com/components/tabs/usage/#secondary-labels)

Maybe it could be worded differently:

Content:
The select components option text is shown as a number, indicating the items per page and the current page. It could display a word instead of a number for the “Items per page” selection.

Modifier:
By default, the select component option text is displayed as a number, showing items per page and the current page. However, we offer an alternative option to use a word instead of a number for the “Items per page” selection if that better meets your requirements.